tests/go/GoVetBearTest.py

```diff
@@ -14,13 +14,19 @@ class GoVetBearTest(LocalBearTestHelper):
     def setUp(self):
         self.section = Section('test section')
         self.uut = GoVetBear(self.section, Queue())
-        self.good_file = os.path.join(os.path.dirname(__file__),
-                                      'test_files',
-                                      'vet_good.go')
-        self.bad_file = os.path.join(os.path.dirname(__file__),
-                                     'test_files',
-                                     'vet_bad.go')
 
-    def test_run(self):
-        self.check_validity(self.uut, [], self.good_file)
-        self.check_invalidity(self.uut, [], self.bad_file)
+    def test_syntax_error(self):
+        good_file = os.path.join(os.path.dirname(__file__),
+                                 'test_files',
+                                 'vet_good.go')
+        bad_file = os.path.join(os.path.dirname(__file__),
+                                'test_files',
+                                'vet_bad_semantics.go')
```

Comment:
In my opinion, it's a bad practice of writing test. Why not using `setUp` to set abs path of test_files and then call it in each test?
I mean you can do below code in setUp
`self.test_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'test_files'))`
And call it in each test by using `good_file = os.path.join(self.test_path, 'vet_good.go')`.

Reply:
Yes, I felt the same. But followed the same practice which was used before. Would update it soon :smile;